docs/cli-reference/gardenadm/gardenadm_token_create.md

```diff
@@ -32,10 +32,12 @@ gardenadm token create
 ### Options
 
 ```
-  -d, --description string                        Description for the bootstrap token (default "Used for joining nodes via `gardenadm join` or connecting autonomous shoots via `gardenadm connect`")
+  -d, --description gardenadm join                Description for the bootstrap token used for gardenadm join
   -h, --help                                      help for create
   -c, --print-connect-command gardenadm connect   Instead of only printing the token, print the full machine-readable gardenadm connect command that can be ran on a machine of a cluster that should be connected to Gardener
   -j, --print-join-command gardenadm join         Instead of only printing the token, print the full machine-readable gardenadm join command that can be copied and ran on a machine that should join the cluster
+      --shoot-name gardenadm connect              Name of the Shoot which should be connected to Gardener via gardenadm connect with this bootstrap token
+      --shoot-namespace gardenadm connect         Namespace of the Shoot which should be connected to Gardener via gardenadm connect with this bootstrap token
```

Comment:
Similar to the above I find it difficult to have the `gardenadm` command listed directly after the option. Maybe it is more clear if we add a separate column, i.e. short option, long option, scenario, description.
WDYT?